inttest/common/footloosesuite.go

```diff
@@ -189,6 +189,13 @@ func (s *FootlooseSuite) keepEnvironment() bool {
 	}
 }
 
+func k0sConfigFlag() string {
+	if os.Getenv("K0S_CONFIG") != "" {
+		return "--config /etc/k0s.conf"
```

Comment:
Always `/etc/k0s.conf`? I would expect it to use whatever path it finds in `K0S_CONFIG`.